samigo/samigo-app/src/webapp/jsf/questionpool/solution.jsp

```diff
@@ -33,6 +33,15 @@
 	<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
 <head><%=request.getAttribute("html.head")%>
 <title><h:outputText value="#{questionPoolMessages.sol_q}" /></title>
+
+<script>

```

Comment:
@jesusmmp

Could we implement this avoiding jQuery? jQuery could be removed in the future and I prefer using vanilla JS.